src/boards/include/boards/pimoroni_picosystem.h

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
```

Comment:
Should this be "Copyright (c) 2021 Pimoroni Ltd." ? :shrug:

Reply:
Not got a clue! Both are valid, but it's on your repo, so...